examples/CSharp/CSharp.Examples/Scenarios/DataFeed.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NBomber;
+using NBomber.Contracts;
+using NBomber.CSharp;
+using Serilog;
+
+namespace CSharp.Examples
+{
+    class DataFeedScenario
+    {
+        public static void Run()
+        {
+            var feed =
+                Feed
+                    .Circular("numbers", Enumerable.Range(0, 10))
+                    .Select(x => (x + 1) * 10)
+                    .Select(x => x.ToString());
+
+            var step = Step.Create("step", async context =>
+            {
+                // you can do any logic here: go to http, websocket etc
+
+                await Task.Delay(TimeSpan.FromSeconds(0.1));
+                var number = (string)context.Data["numbers"];
```

Comment:
it would be great to get a typed version of the feed :)
btw would you mind to add some prefix "feed" like context.Data["feed.numbers"] ?

Reply:
it would be great for sure, but I'm afraid it means adding a type parameter to many types Scenario, ScenarioContext, StepContext. And if feed is not used (evtl. most cases?) it will be just obj. Don't see any easier option currently.

Reply:
yep, agree.

Reply:
@vilinski 
btw would you mind to add some prefix "feed" like context.Data["feed.numbers"] ?

Reply:
ok, 
Now it is just the name of the feed. Not sure wheter it is not less intuitive with prefix.
In any case the usage should be documented

Reply:
@vilinski btw, how this "feed" feature will intersect with passing response between steps?
https://github.com/PragmaticFlow/NBomber.Http/blob/dev/examples/NBomber.Http.CSharp/SequentialSteps.cs#L15

Reply:
Ah I see, since it's now a Dictionary type, then it's fine. I think now you can see why prefix "feed." is kind of appropriate.

Reply:
kind of yes, your point is valid. Bit I'm afraid it's now less obvious what to use as a key. and one can still owerwrite it in the steps anyway

Reply:
yes, I also thought about it that maybe feed should be defined as:
context.FeedData: IReadOnlyDict<>